_layouts/feed.xml

```diff
@@ -0,0 +1,40 @@
+---
+---
+<?xml version="1.0" encoding="UTF-8"?>
+<feed xmlns="http://www.w3.org/2005/Atom" xml:lang="{{ page.locale }}">
+  <id>{{ page.url | absolute_url | xml_escape }}</id>
+  <title>{{ site.data.l10n[page.locale].t.site_title | xml_escape }}</title>
+  <updated>{{ site.time | date_to_xmlschema }}</updated>
+
+  <generator uri="https://jekyllrb.com/" version="{{ jekyll.version }}">Jekyll</generator>
+  <link href="{{ page.url | absolute_url }}" rel="self" type="application/atom+xml"/>
+  <link href="{{ '/' | absolute_url }}" rel="alternate" type="text/html"/>
+  <subtitle>{{ site.data.l10n[page.locale].t.site_subtitle | xml_escape }}</subtitle>
+  <icon>{{ site.feed.icon }}</icon>
+  <logo>{{ site.feed.logo }}</logo>
+
+  <author>
+    <name>{{ site.author | xml_escape }}</name>
+  </author>
+
+  {%- assign posts = site.posts | where_exp:"post", "post.locale == page.locale" | limit:site.feed.limit -%}
+  {% for post in posts %}
```

Comment:
It would be nice to have in feed also software catalog, thoughts @libremente

Reply:
Yes @sebbalex that would be really nice!